app/src/main/java/mozilla/lockbox/view/ItemDetailFragment.kt

```diff
@@ -90,8 +90,17 @@ class ItemDetailFragment : BackableFragment(), ItemDetailView {
         inputLayoutPassword.isHintAnimationEnabled = false
 
         inputUsername.readOnly = true
-        inputUsername.isClickable = true
-        inputUsername.isFocusable = true
+
+        if (item.username.isNullOrEmpty()) {
+            view!!.btnUsernameCopy.setColorFilter(resources.getColor(R.color.white_60_percent))
+            view!!.isClickable = false
+            view!!.isFocusable = false
+            inputUsername.setText(" ", TextView.BufferType.NORMAL)
```

Comment:
does the empty string not work here?

Reply:
nope, no context to find the `getString`

Reply:
can you not use `resources.getString(id: Int)`?